OperationOutcome/England-OperationOutcome-PatientDetailsValidation-Example.xml

```diff
@@ -0,0 +1,27 @@
+<OperationOutcome xmlns="http://hl7.org/fhir">
+    <id value="England-OperationOutcome-PatientDetailsValidation-Example" />
+    <meta>
+        <lastUpdated value="2021-04-21T16:58:00+00:00" />
+    </meta>
+    <issue>
+        <severity value="information" />
+        <code value="processing" />
+        <diagnostics value="This element does not match any known slice defined in the profile https://fhir.nhs.uk/StructureDefinition/NHSDigital-Patient" />
```

Comment:
is this the right URL now, or will this be updated when the patient profile is?

Reply:
No. That escaped into it. Glad your eagle eye caught it. I have amended it and further updated the ValueSet and CodeSystem to conform to NHSE naming convention.